p2p/peer.go

```diff
@@ -338,15 +462,7 @@ func (p *peer) GetRemovalFailed() bool {
 
 // RemoteAddr returns peer's remote network address.
 func (p *peer) RemoteAddr() net.Addr {
-	return p.peerConn.conn.RemoteAddr()
-}
-
-// CanSend returns true if the send queue is not full, false otherwise.
-func (p *peer) CanSend(chID byte) bool {
```

Comment:
Again, for my internal documentation, this was not used in release branches, it is legacy code.